modules/table/src/testFixtures/java/org/apache/ignite/distributed/ItTxTestCluster.java

```diff
@@ -918,7 +919,8 @@ private RaftGroupListener getOrCreateAndPopulateRaftGroupListener(
                     mock(MinimumRequiredTimeCollectorService.class),
                     partitionOperationsExecutor,
                     placementDriver,
-                    clockServices.get(assignment)
+                    clockServices.get(assignment),
+                    new TablePartitionId(tableId, partId)
```

Comment:
Why it's not dependent on whether colocation is enabled or not?

Reply:
It is dependent, this is an `else` branch of that exact check